### PR TITLE
docs: use Markdown table

### LINKED ISSUE #40: Branch to Create a Markdown table and see how GitHub Pages renders the website.
> > Not sure if this is 100% needed. Would be better if we could use a [Markdown table](https://www.markdownguide.org/extended-syntax/).
> 
> Can you make a branch where you add a markdown table and we see how github pages deploys the website? @Panquesito7

It'd be best to create an issue so that everyone can discuss about it and work on it as well. 🙂

_Originally posted by @Panquesito7 in https://github.com/The-Lively-Developers-Community/introductions/issues/35#issuecomment-1541317670_

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Use a Markdown table.
  - We can add contributing guidelines now that the process is a bit different. 🙂

### CODE REVIEW

File: README.md
Comment (KendallDoesCoding) [inline README.md]:
```suggestion
| Name     |      Bio      | Socials   |
```

Let people list any of their socials, either YouTube, Twitter, LinkedIn, Instagram, Discord, etc.

**Review by KendallDoesCoding — APPROVED** [review]
LGTM ✅

**Comment by KendallDoesCoding** [comment]
@The-Lively-Developers-Community/maintainers - please review.

**Comment by KendallDoesCoding** [comment]
@Panquesito7 - this would also fix #40 , right?

**Comment by Panquesito7** [comment]
> @Panquesito7 - this would also fix #40 , right?

Correct. 🙂

File: README.md
Comment (Panquesito7) [reply README.md]: Sounds good! Only one can be linked, though, otherwise, we'd have to add more fields. :)

Comment (KendallDoesCoding) [reply README.md]: Ok, then let's state that in CONTRIBUTING GUIDELINES.

Comment (Panquesito7) [reply README.md]: Yep, we might want to create a file after merging this. 🙂

**Comment by Panquesito7** [comment]
Any update on this?

**Comment by KendallDoesCoding** [comment]
@Panquesito7 - sorry, I forgot about this. Can you please add contributing guidelines to this PR as it will be related to this change and then we'll merge this?

**Comment by KendallDoesCoding** [comment]
> @Panquesito7 - sorry, I forgot about this. Can you please add contributing guidelines to this PR as it will be related to this change and then we'll merge this?

oh, let's make the file after this is merged.

**Pull request merged**
